apps/backend/src/modules/users/user.repository.ts

```diff
@@ -59,8 +68,17 @@ class UserRepository implements Repository {
 		};
 	}
 
-	public async findByEmail(email: string): Promise<null | UserEntity> {
-		const user = await this.userModel.query().findOne({ email });
+	public async findByEmail(
```

Comment:
Oh now I see why it was without deletedAt filter, looks like for now we only need to use this method to find users including deleted, so let's remove includeDeleted parameter and return it as it was

Reply:
Just to clarify, return to two separate methods? Because we need to find users without deleted ones in sign-in flow

Reply:
then you did it right, let's just rename the parameter